recipe/meta.yaml

```diff
@@ -73,9 +73,11 @@ requirements:
 test:
   commands:
     - test -f $PREFIX/lib/libfltk.a  # [unix]
+    - test -f $PREFIX/lib/libfltk${SHLIB_EXT}  # [unix]
     - test -f $PREFIX/lib/libfltk_gl.a  # [unix]
-    - if not exist %LIBRARY_LIB%\\fltk.lib exit 1  # [win]
-    - if not exist %LIBRARY_LIB%\\fltk_gl.lib exit 1  # [win]
+    - if not exist %LIBRARY_LIB%\\fltk_static.lib exit 1  # [win]
+    - if not exist %LIBRARY_LIB%\\fltk_gl_static.lib exit 1  # [win]
```

Comment:
```suggestion
    - if not exist %LIBRARY_LIB%\\fltk.lib exit 1  # [win]
    - if not exist %LIBRARY_LIB%\\fltk_gl.lib exit 1  # [win]
    - if not exist %LIBRARY_LIB%\\libfltk.lib exit 1  # [win]
    - if not exist %LIBRARY_LIB%\\libfltk_gl.lib exit 1  # [win]
```

Reply:
libfltk_gl.lib does not exist I think

Reply:
Ah, okay. Can you check that those files exist? (It looks like only `libfltk` has a shared library`)

Reply:
Should I move bin/libfltk.dll to lib/libfltk.dll ?

Reply:
Nope. `dll`s belong in `bin`